docs/pages/application-access/guides/tcp.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Use Teleport for accessing plain TCP apps
+description: How to configure Teleport for accessing plain TCP apps.
+---
+
+# TCP Apps Access (Preview)
+
+Teleport can provide access to any TCP-based application. This allows users to
+connect to applications which Teleport doesn't natively support such as SMTP
+servers or databases not yet natively supported in Database Access.
+
+## Prerequisites
+
+You will need a running Teleport cluster, either self-hosted or in Teleport Cloud.
+We'll assume that you followed the [Getting Started](../getting-started.mdx)
+guide or the general [App Access Usage](./connecting-apps.mdx) guide to install
+Teleport and setup an Application Service.
+
+## Step 1/3. Configure TCP app
+
+Include your TCP application in the `teleport.yaml` configuration of Teleport
+Application Service:
+
+```yaml
+app_service:
+  enabled: "yes"
+  apps:
+  - name: "tcp-app"
+    uri: tcp://localhost:9876
+    labels:
+      app: tcp-app
+```
+
+Note that the URI scheme must be `tcp://` in order for Teleport to recognize
+this as a TCP application.
+
+You can also register the application [dynamically](./dynamic-registration.mdx)
```

Comment:
I think we can either remove this sentence or add it to a Details box that's closed by default. For this sentence to be meaningful, the user needs to visit the Dynamic Registration guide, read through it, and work out how to convert the instructions above into a dynamic registration, taking them away from the flow of this guide.